internal/server/models/access_key.go

```diff
@@ -21,15 +21,15 @@ type AccessKey struct {
 	Name string `gorm:"uniqueIndex:idx_access_keys_name,where:deleted_at is NULL"`
 	// IssuedFor is the ID of the user that this access key was created for
 	IssuedFor         uid.ID
-	IssuedForIdentity *Identity `gorm:"foreignKey:IssuedFor"`
+	IssuedForIdentity *Identity `gorm:"foreignKey:IssuedFor" db:"-"`
 	ProviderID        uid.ID
 
 	ExpiresAt         time.Time
 	Extension         time.Duration // how long to increase the lifetime extension deadline by
 	ExtensionDeadline time.Time
 
 	KeyID          string `gorm:"<-;uniqueIndex:idx_access_keys_key_id,where:deleted_at is NULL"`
-	Secret         string `gorm:"-"`
+	Secret         string `gorm:"-" db:"-"`
```

Comment:
What does the `db:"-"` tag do? It looks like this explicitly not added as a column in `data/access_key.go`

Reply:
Good question. These are left over from the code gen. I think if we don't use code gen I'll add tests that validate the methods on the tables, and these struct tags will be used by the tests. Maybe we'll add the validation tests either way.

This `db:"-"` indicates to the code gen and to the validation that the field is not mirrored by a database column. It's used for some other purpose.